receiver/k8sclusterreceiver/watcher_test.go

```diff
@@ -479,10 +482,15 @@ func TestObjMetadata(t *testing.T) {
 					ResourceIDKey: "k8s.node.uid",
 					ResourceID:    "test-node-1-uid",
 					Metadata: map[string]string{
-						"foo":                     "bar",
-						"foo1":                    "",
-						"k8s.node.name":           "test-node-1",
-						"node.creation_timestamp": "0001-01-01T00:00:00Z",
+						"foo":                                    "bar",
+						"foo1":                                   "",
+						"k8s.node.name":                          "test-node-1",
+						"node.creation_timestamp":                "0001-01-01T00:00:00Z",
+						"k8s.node.condition_disk_pressure":       "false",
```

Comment:
Shouldn't those first go through Semantic Conventions?

We have been blocking PRs adding new metrics or attributes in k8s components asking for SemConv definition first. Not sure if this has changed.

Reply:
These are experimental entity attributes. They are aligned with the existing metrics. It can be easily changed later since it's experimental

Reply:
Do we want to define such attributes in SemConv first even if we don't have entity signal yet?

Reply:
These won't be eligible as resource attributes due to values being mutable. And I tried to align with our existing stuff. But if we are ready to start with enitities in semantic conventions then I'd be happy to add these.

Reply:
Maybe those would make sense to be defined in the attributes' registry first? (It's a chicken-egg problem)

Having a specific guideline about what we "allow" to be added directly and what not would help here. I'm not sure if we have sth like this right now. I guess we haven't.

Since entity signal is not yet defined maybe it would be just fine if we merge them and file an issue against SemConv to keep track of them making clear the reason why those were added to the receiver now. This issue could be used for reference in the future if there are more requests for additional attributes?

Reply:
> Since entity signal is not yet defined maybe it would be just fine if we merge them and file an issue against SemConv to keep track of them making clear the reason why those were added to the receiver now. This issue could be used for reference in the future if there are more requests for additional attributes?

Your idea sounds good! This will help us make progress on the experimental entities (and hopeful encourage its usage) while still tracking the new additions. I'll start an issue on SemConv.

Reply:
Sounds good to me. Let's have a tracking issue for k8s entity attributes that we have in the collector. The issue will be essentially blocked until we have the signal ready.

Reply:
https://github.com/open-telemetry/semantic-conventions/issues/1693

Reply:
Thank's!